ghcide/src/Development/IDE/Core/Shake.hs

```diff
@@ -130,11 +130,8 @@ import           Development.IDE.GHC.Compat             (NameCache,
                                                          NameCacheUpdater (..),
                                                          initNameCache,
                                                          knownKeyNames,
-#if !MIN_VERSION_ghc(9,3,0)
```

Comment:
:warning: Need to take a look why this condition was dropped

Reply:
Converting to draft because of this

Reply:
https://github.com/haskell/haskell-language-server/pull/3171/commits/e134351387792feb35936940da188b0210866588 merged, resolves some of the issues